### DISCUSSION TITLE
Let the user know when notifications are being fetched

### DISCUSSION THREAD

**Discussion by ilmotta** [status-im/status-legacy#16655] [General]
# Feature Issue

### User Story

>As I user
>I want to know when notifications are loading
>So that even in slower connection scenarios I know the app is working properly

### Notes

- Pull to refresh might not be necessary to implement, due to the fact that the AC state is already kept up-to-date using a polling strategy (see namespace `transport.message.core`).
- Skeletons are probably the way to go, but this should be double-checked with the design team.
- Animations are also open for design analysis, for example, when a new notification arrives and is inserted at the top.

**Comment by cammellos** [comment]
I will move this to a discussion since it requires design input
